css_02/css_02.css

```diff
@@ -0,0 +1,122 @@
+html *
+{
+   font-family: Arial, Helvetica, sans-serif !important;
```

Comment:
Most of the time it is better not to use `!important`. You may want to enhance your knowledge on the topic by reading and understanding deeper the meaning and calculation of css specificity: https://css-tricks.com/specifics-on-css-specificity/ Please achieve the same effect without using `!important`